app/notifications/notifications_ses_callback.py

```diff
@@ -96,14 +99,12 @@ def process_ses_response():
 
             create_outcome_notification_statistic_tasks(notification)
 
-            return jsonify(
-                result="success", message="SES callback succeeded"
-            ), 200
+            return [], 200, {'result': "success", 'message': "SES callback succeeded"}
```

Comment:
so, there are two states here:

1. Success. Neither SES or celery care about any response, but we should make sure the request to SES returns 200.

1. Failure. Neither SES or celery care about any response, but we should make sure we log the reason appropriately. Not sure anyone cares about the response code

So I think with these, we can get away with success returning `None`, and error states returning just the error message

Then, in the celery function, we can say

```python
resp = process_ses_response(...)
if resp:
    logger.error(resp)
```

or something